src/columnsTypes/types/Label/Component.jsx

```diff
@@ -127,6 +130,9 @@ export const LabelsChooser = ({
           label.title.toLowerCase().includes(inputValue.toLowerCase())
         );
 
+  const isNotExists = (label) =>
```

Comment:
maybe call this `tableLabelDoesNotExist`? also, can it go outside of the component definition?